### DISCUSSION TITLE
no search results

### DISCUSSION THREAD

**Discussion by stableartai** [zanllp/infinite-image-browsing#668] [Q&A]
installed the latest rev. can only search in default folders, any search on outputs in alternate location out side the default SD path reveals no results n matter what is selected in filters.

**Comment by stableartai** [comment]
did find the version number on the default empty start page. But same issue unable to search on external output folders. Maybe due to  the walk mode which only allows from default area of SD outputs. As such normal/fixed mode does not work with search

**Comment by zanllp** [comment]
How did you search? Did you enter a Search scope parameter when searching? Could you take a screenshot?

 The prerequisite for searching is that the target folder needs to be added to the search scan path. Refer to the image below, the paths within the red circle are those that will be scanned and indexed.. And it will recursively search the given path.
<img width="1911" alt="image" src="https://github.com/zanllp/sd-webui-infinite-image-browsing/assets/25872019/cea5ee5a-82e6-4c2e-ae6d-37a482d2a250">



 Check if there are any error messages output in the terminal/console/log.log. Screenshot!
